output/example/graphs.json

```diff
@@ -54,23 +54,23 @@
     ],
     "ex_hat_std": [
         [
-            0.479269,
+            NaN,
             0.0
         ],
         [
-            5.86522,
-            0.802755
```

Comment:
those should not be replaced

Reply:
Well spotted! I missed them among all the good changes. This is fixed now along with the other unwanted NaNs.